### PR TITLE
[IMP] theme_*: adapt themes with new `s_company_team` design

### PR DESCRIPTION
**Pull request by anso-odoo**
*: (anelusia, beauty, bewise, graphene, notes, yes)

This PR adapts the design of `s_company_team` for multiple themes, based on the new Odoo 18 snippet redesign.

task-3657640

Requires:
- https://github.com/odoo/odoo/pull/172719

### CODE REVIEW

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

File: theme_graphene/views/customizations.xml [hunk 1]
```diff
@@ -135,29 +135,21 @@
         <attribute name="class" add="pb64" remove="pb48" separator=" "/>
     </xpath>
     <xpath expr="//div[hasclass('s_nb_column_fixed')]/div//h4" position="replace" mode="inner">
-        Tony Fred, <small><b>CEO</b></small>
-    </xpath>
-    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div//p" position="replace">
-        <small>Founder and chief visionary, Tony is the driving force behind the company. He loves to keep his hands full by participating in the development of the software, marketing, and customer experience strategies.</small>
+        Tony Fred, <small><b>CTO</b></small>
```
Comment (bvr-odoo) [inline theme_graphene/views/customizations.xml:138]: - CTO => **CEO** ;)

- Also, `<small>` and `<b>` seem to have no effect ...

image

- Why not use a subtitle as in the default snippet?

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
Small change to do here, thanks for the work :+1:

File: theme_graphene/views/customizations.xml (hunk 1 above)
Comment (anso-odoo) [reply theme_graphene/views/customizations.xml]: Nice catch. I removed the customisations for this one as they are not really needed. Thanks !

**Comment by bvr-odoo** [comment]
@anso-odoo Runbot is red (Upgrade). Could you rebase please :pray:

**Comment by anso-odoo** [comment]
@bvr-odoo should be fine now 👍 thanks !

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**